Quiz-App/js/chemistry.js

```diff
@@ -307,27 +307,27 @@ function res () {
     else incorrect++;
   }
   if (marks <= 4) {
-    document.getElementById("result").style.backgroundColor = "red";
+    document.getElementById("result").style.color = "red";
```

Comment:
previously it was showing a block of color (red,yellow and green) respectively which was looking a bit odd with respect to the background image but now only the text is getting colored which is looking better